packages/devcmd/src/process/ProcessExecutor.ts

```diff
@@ -43,6 +55,20 @@ export class ProcessExecutor {
   /**
    * Executes a process and throws an exception if the exit code is non-zero.
    * Outputs (stdout/stderr) of the process are sent to our stdout/stderr.
+   * 
+   * @param {ProcessInfo} processInfo Information about the process that should be started
```

Comment:
minor: in phrases like this, i find it slightly more readable to say "...the process to start" than "...that should be started" (furthermore, consistently using "execute" instead of "start" here would also be nice)

Reply:
Using `start` for comments. 
Note: the logs say "Starting", should we change that to "Executing" aswell?

Reply:
I think in the logs it's fine (because these logs are user-facing, while the methods and docs here are dev-facing)